overrides/partials/copyright.html

```diff
@@ -0,0 +1,14 @@
+{#-
+  This file was automatically generated - do not edit
```

Comment:
By whom? Does the call not to edit hold true? It's edited as far as I can see.

Reply:
I've removed this file. I've tested, and there is no need to add this file to PMM project